src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILInfo.cs

```diff
@@ -0,0 +1,77 @@
+// Licensed to the.NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Reflection.Emit
+{
+    public class DynamicILInfo
+    {
```

Comment:
Make sure that this has private constructor - like the full version.

Reply:
There's just actually a single internal constructor:
https://github.com/dotnet/coreclr/blob/908891ce9df8d94f28ec6a53a012d39d00c8e65d/src/System.Private.CoreLib/src/System/Reflection/Emit/DynamicILGenerator.cs#L879-L887

Reply:
Note that the constructor here can be without any arguments.